### PR TITLE
add expected result type for SqlStatement

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: src/test/java/com/hazelcast/jdbc/DriverJetTest.java [hunk 1]
```diff
@@ -37,6 +38,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
```
Comment (pivovarit) [inline src/test/java/com/hazelcast/jdbc/DriverJetTest.java:41]: Why is this one getting disabled?

**Review by pivovarit — APPROVED** [review]
Looks good to me. Just one question.

File: src/test/java/com/hazelcast/jdbc/DriverJetTest.java (hunk 1 above)
Comment (SeriyBg) [reply src/test/java/com/hazelcast/jdbc/DriverJetTest.java:41]: This is the same issue as I've described here: https://github.com/hazelcast/hazelcast-jdbc/pull/20#issuecomment-753859825

Comment (pivovarit) [reply src/test/java/com/hazelcast/jdbc/DriverJetTest.java:41]: Good to go then

**Review by devozerov — APPROVED** [review]

**Pull request merged**